Importing gg AND glfw needs importing gl too, or there is a compile error

**V version:** 0.1.16
**OS:** Ubuntu 16.04.3 LTS

**What did you do?**
```
0[18:33:44]/v/misc $ cat gg_and_glfw.v
import gg
import glfw
0[18:33:46]/v/misc $ v gg_and_glfw.v
In file included from .gg_and_glfw.c:99:0:
/v/v/thirdparty/glad/glad.h:26:2: error: #error OpenGL header already included, remove this include, glad already provides it
 #error OpenGL header already included, remove this include, glad already provides it
  ^
V panic: clang error
1[18:33:48]/v/misc $ 
```

**What did you expect to see?**
I expected the code to compile without errors.

 
**What did you see instead?**

The generated C file could not be compiled. It had these lines at the point of the compiler error:
#include <GLFW/glfw3.h>
#include "glad.h"

--- a/vlib/glfw/glfw.v
+++ b/vlib/glfw/glfw.v
@@ -4,6 +4,8 @@
 
 module glfw
 
+import gl
+
 #flag -I @VROOT/thirdparty/glfw 
 #flag -L @VROOT/thirdparty/glfw 
 

